README.rst

```diff
@@ -10,3 +10,20 @@ Coaster: common patterns for Flask apps
 
 Coaster contains functions and db models for recurring patterns in Flask
 apps. Documentation at http://coaster.readthedocs.org/
+
+### Run tests
+
+You can also look at `.travis.yml` for instructions on how to run tests.
+Create a test Coaster DB:
+
+   $ createuser coaster
+   $ createdb -O coaster coaster_test
+
+Ensure you have PySQLite installed before you run:
+
+   $ coverage run `which nosetests`
```

Comment:
Just `nosetests` is sufficient. Coverage is for getting code-coverage reports, an advanced use case for someone looking for code that isn't yet tested.